ui/models/push.js

```diff
@@ -137,15 +143,25 @@ export default class PushModel {
     );
   }
 
-  static getDecisionTaskId(pushId) {
-    const map = PushModel.getDecisionTaskMap([pushId]);
+  static async getDecisionTaskId(pushId, notify) {
+    const taskIdMap = await PushModel.getDecisionTaskMap([pushId], notify);
 
-    return map[pushId];
+    return taskIdMap[pushId];
   }
 
-  static getDecisionTaskMap(pushIds) {
-    return getData(
+  static async getDecisionTaskMap(pushIds, notify) {
+    const { data, failureStatus } = await getData(
       getProjectUrl(`${pushEndpoint}decisiontask/?push_ids=${pushIds}`),
     );
+
+    if (failureStatus) {
+      const msg = `Error getting Gecko Decision Task Ids: ${failureStatus}: ${data}`;
+
+      if (notify) {
+        notify(msg, 'danger', { sticky: true });
+      }
+      throw Error(msg);
```

Comment:
Should there be a return here and above on line 161? If you return data when there's been a failure, it'll contain the original error message.